libraries/AP_Proximity/AP_Proximity.cpp

```diff
@@ -282,9 +281,9 @@ void AP_Proximity::detect_instance(uint8_t instance)
     case Type::None:
         return;
     case Type::SF40C_v09:
-        if (AP_Proximity_LightWareSF40C_v09::detect()) {
+        if (AP_Proximity_LightWareSF40C::detect()) {
             state[instance].instance = instance;
-            drivers[instance] = new AP_Proximity_LightWareSF40C_v09(*this, state[instance]);
+            drivers[instance] = new AP_Proximity_LightWareSF40C(*this, state[instance]);
             return;
         }
         break;
```

Comment:
If we are removing the driver completely we should remove the `case Type::SF40C_v09:` completely.

For the enumeration in `AP_Proximty.h` we can do something like the following to let others know they cannot use that slot aka Type = 1,
```cpp
    enum class Type {
        None    = 0,
        // 1 was SF40C_v09
```

Reply:
Thanks, @rmackay9 sir for review and @hendjoshsr71 sir for help. I have done suggested changes, please have a look.